gitpg/williams/langtonant.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
```

Comment:
Also, please follow instructions under katas/ folder, this solution should be under katas/<kata>/<your name> folder